server/chat-plugins/scavenger-games.ts

```diff
@@ -370,7 +377,7 @@ const TWISTS: {[k: string]: Twist} = {
 		desc: 'After completing the hunt, players will guess what the most common incorrect answer for each question is.',
 		onAfterLoad() {
 			this.guesses = {};
-			this.incorrect = new Array(this.questions.length).fill('').map(() => ({}));
+			this.incorrect = Array.from({length: this.questions.length}).map(() => ({}));
```

Comment:
Why change this? The other way looks cleaner.

Reply:
I thought that the convention was to prefer Array.from instead of Array(X); is that not the case?

Reply:
That being said technically I could just use a this.questions.map(() => ({}))

Reply:
> I thought that the convention was to prefer Array.from instead of Array(X); is that not the case?

Not to my knowledge. I'd favor either `Array(num)` or `questions.map(() => {})`, up to you.